### ISSUE TITLE
Yara 3.7.1 issue with Androguard

### ISSUE THREAD

**Issue by rajeshkumar91** [Koodous/androguard-yara#10]
Hi,

I followed all the steps mentioned in Readme.txt for adding androguard module and recompiled my yara(version 3.7.1)

After that, I used the example mentioned in the Readme.txt.

python download_androguard_report.py -s d8adb784d08a951ebacf2491442cf90d21c20192085e44d1cd22e2b6bdd4ef5f

And downloaded the report and also downloaded the respective sample and pointed to the test rule file given in the repo,

yara -x androguard=d8adb784d08a951ebacf2491442cf90d21c20192085e44d1cd22e2b6bdd4ef5f-report.json test/rule.yar d8adb784d08a951ebacf2491442cf90d21c20192085e44d1cd22e2b6bdd4ef5f

I got the following error:

test/rule.yar(1): error: unknown module "androguard"
test/rule.yar(1): error: invalid field name "package_name"

Kindly help me with this issue

**Comment by cgupta6** [comment]
Same issue
please help

**Comment by plutec** [comment]
Did you compile yara with cuckoo module enabled? (

**Comment by rajeshkumar91** [comment]
Yes, compiled with cuckoo module enabled. Also tried in two different systems and compiled yara several number of times. The mentioned issue happened every time.

**Comment by cgupta6** [comment]
I am also facing and have tried with cuckoo enabled
